connector_prestashop/backend.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import openerp.addons.connector.backend as backend
+
+
+prestashop = backend.Backend('prestashop')
+# version < 1.6.0.9
+prestashop1500 = backend.Backend(parent=prestashop, version='1.5')
+# version 1.6.0.9 - 1.6.0.10
+prestashop1609 = backend.Backend(parent=prestashop, version='1.6.0.9')
+# version >= 1.6.0.11
+prestashop16011 = backend.Backend(parent=prestashop, version='1.6.0.11')
```

Comment:
Is there a difference in the PrestaShop's API between 1.6.0.9 and 1.6.0.11 that justify to have different backends?